compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -426,6 +428,7 @@ internal fun ProvideContainerCompositionLocals(
     content: @Composable () -> Unit,
 ) = with(composeContainer) {
     CompositionLocalProvider(
+        LocalHapticFeedback provides HapticFeedback(),
```

Comment:
It will be recreated on each recomposition

Reply:
Right, moved it to `ComposeContainer`.